keyboards/ut472/keymaps/praxling/readme.md

```diff
@@ -0,0 +1,14 @@
+# praxling's modified layout
+
+This has a single alpha layout for Dvorak, with the locations of the semicolon and single quote swapped to match the modified Programmer Dvorak layout (http://programmer-dvorak.appspot.com/)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make ut472:praxling-dvorak
+
+* Download or Clone QMK Firmware: <https://github.com/qmk/qmk_firmware/>
+* Extract QMK Firmware to a location on your hard drive
+* Copy this folder into %s
+* You are now ready to compile or use your keymap with the source
```

Comment:
I'm ill at ease with this section. Once your pull request is merged, anyone who downloads or clones QMK Firmware is going to already have this folder in the right location.